shapely/libgeos.py

```diff
@@ -0,0 +1,116 @@
+"""
+Proxies for the GEOS shared library, versions, and configure options
+TODO: find a flexible way to load a dynamic library from a specified path
```

Comment:
Also, I was considering moving `get_geos_config` ([setup.py#L151](https://github.com/Toblerity/Shapely/blob/2e58796977672acfd487607981b4fc2f62b23811/setup.py#L151)) to this module, but was hesitant since it is only a development tool. I'm not sure how `geos-config` can be used to sniff out the dynamic library location. (Somewhere under `geos-config --prefix`?). Is it possible to get `setup.py` to write a configuration file on as to where the dynamic library should always be loaded?